egs/wsj/s5/utils/data/get_uniform_subsegments.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0.
+
+import argparse
+import logging
+import sys
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description="""Get uniform subsegments of segments.""")
```

Comment:
This description is a bit too short, it doesn't really explain it clearly.  Can you please add more documentation with an example, here or in a comment?